configure.ac

```diff
@@ -5415,6 +5415,40 @@ PKG_CHECK_MODULES([LIBLZMA], [liblzma], [have_liblzma=yes], [
   ])
 ])
 
+have_libzstd=no
+AC_DEFUN([TEST_ZSTD_VERSION],[
+  AC_MSG_CHECKING([if libzstd is new enough])
```

Comment:
There seem to be two problems with the `new enough` check:
1. There should be `AC_MSG_RESULT([yes])` and `AC_MSG_RESULT([no])` calls to complete the messages:
```
checking for libzstd... yes
checking if libzstd is new enough... checking for hstrerror... yes

```